module_1/first-react-app/src/App.js

```diff
@@ -26,6 +33,24 @@ function App() {
     alert("Searching movies with name: " + query);
   }
 
+  const handleTileClick = (movie) => {
+    //alert("trying to select movie: ", movie);
+    console.log(movie.movieName);
```

Comment:
I suppose we don't need console.log, please remove

Reply:
Removed console.log() statement